subsys/net/lib/nrf_cloud/Kconfig

```diff
@@ -44,4 +36,10 @@ config NRF_CLOUD_PORT
 config NRF_CLOUD_IPV6
 	bool "Configure nRF Cloud library to use IPv6 addressing. Otherwise IPv4 is used."
 
+module=NRF_CLOUD
+module-dep=LOG
```

Comment:
Should we depend on `LOG` or `NET_LOG` @rlubos ?
I am using the networking subsystem Kconfig template.

Reply:
I'd say to stick to the `LOG` - less modules inbetween means less problems to inherit from upstream. Rember the MQTT, right?